### ISSUE TITLE
Bug Report: Freeplay song box's position gets misaligned after completing a song for the first time

### ISSUE THREAD

**Issue by ManMaster707** [FunkinCrew/Funkin#3267]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

v0.5.0

### Context (Provide images, videos, etc.)

When you get booted back to the freeplay menu after completing a song for the first time, once the rank animation is finished the completed song's box will not scroll along properly with the other boxes, misaligning itself massively. This happens every single time you complete a song for the first time (or get a new rank? I haven't tested that yet). 
Sometimes the song's details (rank, difficulty, song name, bpm...) on the box get misaligned too. 
I'm not sure if spamming the up and down keys during the animation amplifies the effect, but it certainly isn't the cause, since it still happened without pressing any key, at least in the windows build.

Here are some screenshots I took which should help to get a better understanding of what happens. Note the ranks on the boxes!

![Screenshot 2024-09-14 184954](https://github.com/user-attachments/assets/5974801a-e729-4e08-a6c6-7b55594ed248)
![Screenshot 2024-09-14 183123](https://github.com/user-attachments/assets/124cb737-d263-4e83-ae04-3fc6c2dde738)
![Screenshot 2024-09-14 183128](https://github.com/user-attachments/assets/63693e4a-dd3c-4704-8b48-73b855db5456)
![Screenshot 2024-09-14 183134](https://github.com/user-attachments/assets/b5c3385c-405f-43bf-ad72-c1aec0b14a19)
![Screenshot 2024-09-14 183538](https://github.com/user-attachments/assets/b3ad18c4-c5f8-4390-bb32-7989f06c48d4)
![Screenshot 2024-09-14 184403](https://github.com/user-attachments/assets/15abacc5-de6f-4420-ae8f-38d44534fd8b)
![Screenshot 2024-09-14 184725](https://github.com/user-attachments/assets/5640d76e-fd77-4fa7-a627-7cbf2c8f3d00)

This isn't the end of it though.
I've also managed to encounter this glitch but on a song which I didn't play yet, in this case M.I.L.F (not the erect version!).
This time I don't have the slightest clue on how this could have happened, but I've got some screenshots yet again to show the validity of it all.

![Screenshot 2024-09-14 185914](https://github.com/user-attachments/assets/71975ed9-62ad-457c-a156-06e94b940041)
![Screenshot 2024-09-14 185919](https://github.com/user-attachments/assets/f67891d5-784b-42bd-9307-0ee970802ff0)
![Screenshot 2024-09-14 190015](https://github.com/user-attachments/assets/674d98cd-1317-4064-917b-8f4bd4fd5821)

I don't think this bug has been documented yet in detail, but if it was i'll close this issue once informed.

### Steps to reproduce (or crash logs, errors, etc.)

- Complete a song which you haven't played yet (I hypotesize this works if you get a higher rank on it too);
- Once you begin to scroll in the freeplay menu, the bug should be apparent.

**Comment by Hundrec** [comment]
Also reported in #3239
Very useful screenshots!

**Comment by EliteMasterEric** [comment]
Duplicate of #3239, but yes lots of great screenshots!
